doc/user/tutorials/sharing/index.md

```diff
@@ -25,6 +25,12 @@ Creating a User Account
 
 If the person you would like to share data with within IRIDA does not have a user account, you'll need to make an account for them.
 
+You may create a new user account in IRIDA if you have the **Manager** or **Administrator** system role.
+
+After logging in to IRIDA, click on the **Users** menu, and select "Create User":
```

Comment:
In `You may create a new user account in IRIDA if you have the **Manager** or **Administrator** system role` make the Administrator a link to the section in the Administrator guide as there are separate instructions for Admins


Change to:

After logging into IRIDA, click the gear icon in the top right hand side of the navbar and click **Create User**.

With this update the screenshot as well to also have the gear icon outlined